yarn-project/noir-libs/noir-aztec/src/context.nr

```diff
@@ -292,7 +292,7 @@ impl Context {
         contract_address: Field, 
         function_selector: Field, 
         args: [Field; ARGS_COUNT]
-    ) -> [Field; RETURN_VALUES_LENGTH] {
+    ) {
```

Comment:
How do public functions return values?

Reply:
This is only for private-to-public calls (hence the Context object, which only lives on private-land), and there public functions don't return anything since they are not executed synchronously. I was waiting for #1515 to add autogeneration for calling public-from-public, since that's not supported in this version.

Reply:
Ah, yes. Perfect, thanks!